modules/ting/src/Search/TingSearchCommonFields.php

```diff
@@ -22,6 +22,8 @@ class TingSearchCommonFields {
   const LANGUAGE = self::_PREFIX . 'language';
   const SUBJECT = self::_PREFIX . 'subject';
 
+  const ACQUISITION_DATE = self::_PREFIX . 'acquisition_date';
```

Comment:
I do not know exactly where this belongs but adding a description of what these fields are would be good.

Reply:
Hm, yes. And I guess they need to be documented at least 2 times in the current codebase. First a general description in TingSearchCommonFields, and then once more in the providers search_map_common_fields implementation.

I'll start doing it from now.

Reply:
Correction, I will document the general fields for now and then document the provider-fields as we go along.